rca/editorial/models.py

```diff
@@ -0,0 +1,116 @@
+from django.db import models
+from modelcluster.fields import ParentalKey
+from wagtail.admin.edit_handlers import (
+    FieldPanel,
+    InlinePanel,
+    MultiFieldPanel,
+    PageChooserPanel,
+)
+from wagtail.api import APIField
+from wagtail.core.fields import RichTextField
+from wagtail.images.edit_handlers import ImageChooserPanel
+
+from rca.utils.models import BasePage, RelatedPage
+
+
+class EditorialPageRelatedSchoolsAndResearchPages(RelatedPage):
+    source_page = ParentalKey(
+        "EditorialPage", related_name="related_schools_and_research_pages"
+    )
+    panels = [
+        PageChooserPanel("page", ["schools.SchoolPage", "research.ResearchCentrePage"])
+    ]
+
+    api_fields = [APIField("page")]
```

Comment:
Do I need this line? It was used in the ProgrammePage but not sure if it is needed here. From looking online, it looks like this field defines the fields that will be accessible in the API, just not entirely sure what the difference is in this case with/without the field..

Reply:
I'd remove it - we may end up needing it - but for the purposes of this ticket we aren't interested in defining this right now